### DISCUSSION TITLE
Exporter for kafka lag / offset from kafka-ui to another monitoring system

### DISCUSSION THREAD

**Discussion by lukass77** [provectus/kafka-ui#2871] [Ideas]
does there is a support to export metrics from kafka-ui to another monitoring system like datadog / grafana / statsd ..
on top of the mertics value we see on UI .

1. can we use an exsisting rest api to fetch offset / log / lag - values ?
2. can we implement a generic interface which will export metrics to another monitoring system ?

did not see any documentation on this supported functionality ..
Thanks!

**Comment by Haarolean** [comment]
Hey, not yet. Planning to implement this within #942 :)

**Comment by lukass77** [comment]
Hey @Haarolean can you please share if we can have integration with stasd / datadog .. just saw you close this issue ..so in case this is supported .. just update here .. thanks!!

**Reply by Haarolean** [reply]
Hey, I closed the discussion, not the issue.
You can write an exporter yourself, all our API endpoints are available.

**Comment by leo-li-wcq** [comment]
> Hey, I closed the discussion, not the issue. You can write an exporter yourself, all our API endpoints are available.

Are there any docs for these API endpoints?

**Reply by Haarolean** [reply]
@leo-li-wcq https://github.com/provectus/kafka-ui/blob/master/kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml
Also, at the moment, we're able to export metrics in prometheus format

**Reply by leo-li-wcq** [reply]
Thanks for the update @Haarolean, I think the export metrics related to https://github.com/provectus/kafka-ui/issues/942, any idea when this will be rollout?

**Reply by Haarolean** [reply]
@leo-li-wcq https://github.com/provectus/kafka-ui/discussions/4255
